databases/week2/part4.sql

```diff
@@ -0,0 +1,44 @@
+-- Create a new database
+CREATE DATABASE school;
+
+-- Switch to the new database
+USE school;
+
+-- Create the Class table
+CREATE TABLE Class (
+  id INT PRIMARY KEY,
+  name VARCHAR(50) NOT NULL,
+  begins DATE,
+  ends DATE,
+  student_id INT NOT NULL,
+  FOREIGN KEY (student_id) REFERENCES Student(id)
```

Comment:
I recommend reading this article to learn about constraints. These things help to secure the data you're putting into tables. 
https://www.w3schools.com/sql/sql_foreignkey.asp